blueocean-pipeline-editor/src/main/js/services/PipelineSyntaxConverter.js

```diff
@@ -187,6 +187,8 @@ export function convertStageFromJson(topStage: PipelineStage): StageInfo {
             const stage = convertStageFromJson(topStage.parallel[j]);
             topStageInfo.children.push(stage);
         }
+    } else if (topStage.stages) {
+        // TODO: Will need to be filled in when we actually have the ability to do something with sequential stages.
```

Comment:
Should we log a warning in console as well?

Reply:
Done